src/components/EmojiPicker/EmojiPickerMenu/index.tsx

```diff
@@ -141,21 +142,7 @@ function EmojiPickerMenu({onEmojiSelected, activeEmoji}: EmojiPickerMenuProps, r
                 return;
             }
 
-            // Select the currently highlighted emoji if enter is pressed
             if (!isEnterWhileComposition(keyBoardEvent) && keyBoardEvent.key === CONST.KEYBOARD_SHORTCUTS.ENTER.shortcutKey) {
-                let indexToSelect = focusedIndex;
-                if (highlightFirstEmoji) {
-                    indexToSelect = 0;
-                }
-
-                const item = filteredEmojis[indexToSelect];
-                if (!item) {
-                    return;
-                }
-                if ('types' in item || 'name' in item) {
-                    const emoji = typeof preferredSkinTone === 'number' && item?.types?.[preferredSkinTone] ? item?.types?.[preferredSkinTone] : item.code;
-                    onEmojiSelected(emoji, item);
-                }
                 // On web, avoid this Enter default input action; otherwise, it will add a new line in the subsequently focused composer.
                 keyBoardEvent.preventDefault();
```

Comment:
Moving `Enter` login to `useKeyboardShortcut` and Keeping prevent default and stop propagation here cause this issue https://github.com/Expensify/App/issues/50561, and we fixed it here https://github.com/Expensify/App/pull/50971